### PR TITLE
[#IOPID-2411] `GetVisibleServices` removal from OpenApi

### PR DESCRIPTION
**Pull request by salvatorelaiso**
#### List of Changes

- Remove OpenApi `GetVisibleServices` section.
- Remove `GetVisibleServices` logging level value from `host.json`

#### Motivation and Context

The functionality `GetVisibleServices` was removed

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by gquadrati** [comment]
Please, execute a build to cleanup `index.yaml` too

**Comment by salvatorelaiso** [comment]
> Please, execute a build to cleanup `index.yaml` too

Done in [d8c65a2](https://github.com/pagopa/io-functions-app/pull/322/commits/d8c65a26c105d53ce988457329f347a15f60f3a8)

File: openapi/index.yaml [hunk 1]
```diff
@@ -23,24 +23,6 @@ schemes:
...
-      description: >-
-        Returns all the services that have the 'is_visibile' field value set to
-        true.
-      tags:
-        - restricted
-      responses:
-        '200':
-          description: Found.
-          schema:
-            $ref: '#/definitions/PaginatedServiceTupleCollection'
```
Comment (gquadrati) [inline openapi/index.yaml:39]: We can remove `PaginatedServiceTupleCollection` and all related unused definitions as well

**Review by gquadrati — APPROVED** [review]
LGTM
Please @rodoviti, can you check this PR? We removed the `getVisibleServices` endpoint from the openapi

**Review by rodoviti — APPROVED** [review]
LGTM

**Pull request merged**